traits/tests/test_trait_dict_object.py

```diff
@@ -170,6 +201,21 @@ def test_clear(self):
         self.assertEqual(self.changed, {})
         self.assertEqual(self.removed, {"a": 1, "b": 2})
 
+    def test_clear_empty_dictionary(self):
```

Comment:
This seems unrelated, but I may be missing something.

Reply:
It helps if I ready the actual PR description.

Reply:
Yep, sorry; I can break this out into a separate PR if that's helpful.